#!/usr/bin/python3
def raise_exception():
    t = 'fd'
    t += 1
